### PR TITLE
Feature/dashboards

### PR DESCRIPTION
**Pull request by pernstrong**
#### What's this PR do?
This is more or less an MVP of the app. There is still a lot of work to be done with retooling the displays and dashboards but almost all of the functionality is covered. The fetch DELETE request is not working so that needs to be figured out as well
#### Where should the reviewer start?
From the top! index.js and scripts.js or the classes are good places to start.
#### How should this be manually tested?
Login as customer or manager and make a reservation/see the dashboard info.
#### Any background context you want to provide?
n/a
#### What are the relevant tickets?
n/a
#### Screenshots (if appropriate)
n/a
#### Questions:
n/a

### CODE REVIEW

**Comment by pernstrong** [comment]
@hannahhch @thatPamIAm

File: src/User.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      rooms.forEach(room => {
+        if (room.number === booking.roomNumber) {
+          total += room.costPerNight;
+        }
+      })
+      return parseInt(total.toString(), 10)
+    }, 0)
+  }
+
+  findPastBookings(date) {
```
Comment (ttarlov) [inline src/User.js:20]: Very interesting way to find past bookings. I have not thought of this.

File: src/css/_mixins.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
+
+
+@mixin manager-dash() {
```
Comment (ttarlov) [inline src/css/_mixins.scss:3]: nice mixins!

**Review by ttarlov — APPROVED** [review]
Good job man! You did this super fast.

**Pull request merged**